.pre-commit-config.yaml

```diff
@@ -21,7 +21,7 @@ repos:
       - id: trailing-whitespace
         name: Check for trailing whitespaces (auto-fixes)
   - repo: https://github.com/pycqa/isort
-    rev: 5.8.0
+    rev: 5.12.0
```

Comment:
@ali-i3 Is this the correct version? This looks very recent (maybe the latest, before pre-release versions). Does this work with Py3.6?